src/components/ProjectCard/ProjectModal.js

```diff
@@ -81,6 +83,7 @@ const ProjectModal = ({ isOpen, onDismiss, project, addProjectToProjectsPage })
   }
 
   const handleSuccessResponse = (response, languageSuccessMessage) => {
+    refreshCurrentUser() // this ensures the user has the right privaleges to the newly created project
```

Comment:
Remember how our current user was being initialized way too often? That hid this bug. It wasnt from the upgrade.

Reply:
ah ok - I suspected it was something else. Nice job!